exec-api/src/main/java/com/capitalone/dashboard/executive/service/PortfolioServiceImpl.java

```diff
@@ -129,8 +135,18 @@ public PortfolioResponse getPortfolio(String name, String lob) {
      */
     public List<PortfolioResponse> getPortfolios() {
         List<PortfolioResponse> portfolioResponses = new ArrayList<>();
-        List<Portfolio> portfolioList;
-        portfolioList =  portfolioRepository.findAllByOwnersNotNull();
+        List<Portfolio> newportfolioList = new ArrayList<>();
+        List<Portfolio> portfolioList = portfolioRepository.findAllByOwnersNotNull();
+        List<PortfolioThumbnail> portfolioListThumbnail = portfolioRepositoryThumbnail.findAll();
+        for (Portfolio pList:portfolioList) {
+            for (PortfolioThumbnail p1:portfolioListThumbnail) {
+                if (pList.getName().equalsIgnoreCase(p1.getName())) {
```

Comment:
Please consider using StringUtils.equals()

Reply:
We removed findAll() to avoid redundant and updated as findAllByNameAndThumbnail().

Reply:
Updated to StringUtils.